modules/ROOT/pages/errors/gql-errors.adoc

```diff
@@ -432,3 +432,265 @@ Status description:: error: syntax error or access rule violation - auth info ex
+
+
+=== 51N33
+
+Status description:: error: system configuration or operation exception - replication error. This member failed to replicate transaction, try again.
+
+
+=== 51N34
+
+Status description:: error: system configuration or operation exception - write transaction failed due to leader change. Failed to write to the database due to a cluster leader change. Retrying your request at a later time may succeed.
+
+
+=== 51N35
+
+Status description:: error: system configuration or operation exception - database location changed. The location of `{ $db }` has changed while the transaction was running.
+
+
+=== 51N39
+
+Status description:: error: system configuration or operation exception - raft log corrupted. Expected set of files not found on disk. Please restore from backup.
+
+
+=== 51N40
+
+Status description:: error: system configuration or operation exception - unable to start database. Database `{ $db }` failed to start. Try restarting it.
+
+
+=== 51N41
+
+Status description:: error: system configuration or operation exception - admin operation failed. Server or database admin operation not possible.
+
+
+=== 51N43
+
+Status description:: error: system configuration or operation exception - cannot deallocate servers. Cannot deallocate server(s) `{ $serverAndedList }`.
+
+
+=== 51N44
+
+Status description:: error: system configuration or operation exception - cannot drop server. Cannot drop server `{ $server }`.
+
+
+=== 51N45
+
+Status description:: error: system configuration or operation exception - cannot cordon server. Cannot cordon server `{ $server }`.
+
+
+=== 51N46
+
+Status description:: error: system configuration or operation exception - cannot alter server. Cannot alter server `{ $server }`.
+
+
+=== 51N47
+
+Status description:: error: system configuration or operation exception - cannot rename server. Cannot rename server `{ $server }`.
+
+
+=== 51N48
+
+Status description:: error: system configuration or operation exception - cannot enable server. Cannot enable server `{ $server }`.
+
+
+=== 51N49
+
+Status description:: error: system configuration or operation exception - cannot alter database. Cannot alter database `{ $db }`.
+
+
+=== 51N50
+
+Status description:: error: system configuration or operation exception - cannot recreate database. Cannot recreate database `{ $db }`.
+
+
+=== 51N51
+
+Status description:: error: system configuration or operation exception - cannot create database. Cannot create database `{ $db }`.
+
+
+=== 51N52
+
+Status description:: error: system configuration or operation exception - cannot alter database topology. Number of primaries `{ $count }` needs to be at least 1 and may not exceed `{ $upper }`.
+
+
+=== 51N53
+
+Status description:: error: system configuration or operation exception - cannot alter database topology. Number of secondaries `{ $count }` needs to be at least 0 and may not exceed `{ $upper }`.
+
+
```

Comment:
```suggestion
=== 51N52

Status description:: error: system configuration or operation exception - number of primaries out of range. Cannot alter database topology.  Number of primaries `{ $count }` needs to be at least 1 and may not exceed `{ $upper }`.


=== 51N53

Status description:: error: system configuration or operation exception - number of secondaries out of range. Cannot alter database topology. Number of secondaries `{ $count }` needs to be at least 0 and may not exceed `{ $upper }`.

```

We noticed that these two had the same subcondition so they got modified a bit. I have updated the spreadsheet too now.